tests/src/queue/tests/mod.rs

```diff
@@ -0,0 +1,21 @@
+use fuels::prelude::{abigen, launch_provider_and_get_wallet};
+
+abigen!(Script(
+    name = "TestQueue",
+    abi = "src/queue/out/debug/queue_test-abi.json"
+),);
+
+mod success {
+
+    use super::*;
+
+    #[tokio::test]
+    async fn runs_queue_test_script() {
+        let path_to_bin = "src/queue/out/debug/queue_test.bin";
+        let wallet = launch_provider_and_get_wallet().await;
+
+        let instance = TestQueue::new(wallet, path_to_bin);
+
+        let _result = instance.main().call().await;
```

Comment:
This should `assert!(<return_value>)` otherwise this does not test anything aside from that we do not revert.

Reply:
Removed Rust tests